.travis.yml

```diff
@@ -8,6 +8,20 @@ env:
     - ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe
     - ORACLE_SID=XE
 
+matrix:
+  include:
```

Comment:
This structure results in two jobs on the build screen, and it isnt possible to determine what each job is doing.

A fix for this is to add an unused environment variable "APT_ADDON=[0|1]" or similar to explain what each build does.
Any other solutions for that problem?
Otherwise I'll add the env var and resubmit the PR.